testing.md

```diff
@@ -3,52 +3,61 @@
 ## Developing Tests
 
 To write and run the tests, you'll need to set up your testing environment in Python.
-Be forewarned, as part of the requirements,
-you have to install the `docassemble` package, which is complex. If you have trouble downloading that
-package, take a look at the [installation instructions](https://docassemble.org/docs/installation.html)
-or post your question in the [Docassemble Slack channel](docassemble.slack.com).
 
 ### Set up a virtual environment
 
-First, set up a virtual environment called `docassemble` to hold all the packages
-related to this repository.
+First, set up a virtual environment in your command prompt called `venv` to hold all the packages related to this repository.
 This will keep your code clean, and make sure this repository does not interfere with
 your other projects.
 
+Run these lines in your command prompt.
+
 ```
-$ cd docassemble/assemblylinewizard
-$ pip3 install virtualenv
-$ virtualenv -p $(which python3) docassemble
-$ source docassemble/bin/activate
-$ pip install -r requirements.txt
+pip3 install virtualenv
+virtualenv -p $(which python3.8) venv
+source venv/bin/activate
+pip3 install --upgrade pip
+pip3 install -r docassemble/assemblylinewizard/requirements.txt
+pip3 install --editable .
 ```
 
+<!-- I had to run `virtualenv -p /usr/local/opt/python@3.8/bin/python3 venv` for the second line -->
```

Comment:
This is in a comment and doesn't appear in the README. I don't think it was about the version of my OS, I think it was about previous things I'd installed and the links that got created for them which I wasn't able to resolve another way. Before adding it to the visible README instructions, I'd like to discuss with others whether such a warning should be added to the text and, if so, what it should say.

Reply:
I knew it's a comment, by future readers I meant future maintainers. Sorry, that was misleading, :sweat_smile:  I wasn't trying to suggest making part of the visible readme. And it is relevant to what you OS is though: in Ubuntu, mine's just `/usr/bin/python3`.

Reply:
It's also based on what's in the file that specifies your $PATH (forgotten what it's called now). Mine's a total mess. I think it can be based on other configs too. I can add that not to the comment, but until someone can replicate it on the same system with no noise from other configuration settings, I don't think we should make it visible.